lib/utils/websocket.dart

```diff
@@ -38,7 +38,7 @@ class WebSocket {
     // Initiate communication 
     // To Connect to the Localhost in the App, Read this following
     // https://stackoverflow.com/questions/4779963/how-can-i-access-my-localhost-from-my-android-device
-    _channel = IO.io('http://localhost:3000/', <String, dynamic>{
+    _channel = IO.io('http://56d6501f.ngrok.io/', <String, dynamic>{
```

Comment:
@ganeshpatro321 How does this work actually?

Reply:
Read the readme. I have described steps to create a local tunnel. It basically exposes your local server through a tunnel to the internet, and you can call your apis through that link. @aquibbaig

Reply:
Thanks @ganeshpatro321